### PR TITLE
Fill changelog for v0.16

### PR DESCRIPTION
**Pull request by adonath**
**Description**
This PR fills the changelog for Gammapy v0.16. I've summarised the most important bug fixes, API changes and new features.

**Dear reviewer**
Let me know if you have any comments.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,8 +1,116 @@
...
+`model gallery <modeling/gallery/index.rst>`__ was added. A general introduction
+on how to work with the different models is now avilable in a dedicated `models tutorial <notebooks/models.html>`__.
+The spectral analysis of an extended source is demonstrated in the newly
+added `extended source spectral analysis tutorial <notebooks/extended_source_spectral_analysis.ipynb>`__.
+
+The ``coordsys`` attribute of ``WcsGeom`` and ``HpxGeom`` was
+renamed to ``frame`` and now supports arbitrary Astropy coordinate
+frames.
+
+The ``Datasets`` and ``Models`` container objects now required unique
```
Comment (registerrier) [inline CHANGES.rst]: now require

File: CHANGES.rst [hunk 2]
```diff
@@ -1,8 +1,116 @@
...
+names of the objects contained. By default unique identifier are generated
+in the model and dataset objects. The ``Datasets``, ``Models`` as well
+as ``Observations`` container classes, were extend to now support
```
Comment (registerrier) [inline CHANGES.rst]: A distinct paragraph for this maybe?

File: CHANGES.rst [hunk 3]
```diff
@@ -1,8 +1,116 @@
...
+in place ``.append()``, ``.extend()`` and ``.insert()`` operations.
+
+For Gammapy v0.16 the API of the ``SensitivityEstimator`` and ``TSMapEstimator``
+was adapted to take a ``MapDataset`` or ``MapDatasetOnOff`` as input.
+The ``ASmooth`` class was renamed to ``ASmoothMapEstimator`` and also
+adapted to work with ``MapDataset`` and ``MapDatasetOnOff``.
+
+To further improve API consistency the ``EnergyDispersion`` class
```
Comment (registerrier) [inline CHANGES.rst]: Why not start with this since this is the change most affecting users?

File: CHANGES.rst [hunk 4]
```diff
@@ -1,8 +1,116 @@
...
+was renamed to ``EDispKernel`` and the ``SkyModels``class was
+renamed to a more general ``Models`` class. Agian this release contains
```
Comment (registerrier) [inline CHANGES.rst]: Again

File: CHANGES.rst [hunk 5]
```diff
@@ -1,8 +1,116 @@
...
++++++++
+
+- Released Jan 31, 2020
+- 8 contrinutors
+- 60 pull requests (not all listed below)
+
+**What's new**
+
+For Gammapy v0.16 a ``FoVBackgroundMaker`` was implemented, which supports
+different methods of adapting the norm and tilt of a filed of view background
```
Comment (Bultako) [inline CHANGES.rst]: field

File: CHANGES.rst [hunk 6]
```diff
@@ -1,8 +1,116 @@
...
+on how to work with the different models is now avilable in a dedicated `models tutorial <notebooks/models.html>`__.
+The spectral analysis of an extended source is demonstrated in the newly
+added `extended source spectral analysis tutorial <notebooks/extended_source_spectral_analysis.ipynb>`__.
+
+The ``coordsys`` attribute of ``WcsGeom`` and ``HpxGeom`` was
+renamed to ``frame`` and now supports arbitrary Astropy coordinate
+frames.
+
+The ``Datasets`` and ``Models`` container objects now required unique
+names of the objects contained. By default unique identifier are generated
```
Comment (Bultako) [inline CHANGES.rst]: identifiers

File: CHANGES.rst (hunk 2 above)
Comment (Bultako) [inline CHANGES.rst]: extended

File: CHANGES.rst [hunk 7]
```diff
@@ -1,8 +1,116 @@
...
+as ``Observations`` container classes, were extend to now support
+in place ``.append()``, ``.extend()`` and ``.insert()`` operations.
+
+For Gammapy v0.16 the API of the ``SensitivityEstimator`` and ``TSMapEstimator``
+was adapted to take a ``MapDataset`` or ``MapDatasetOnOff`` as input.
+The ``ASmooth`` class was renamed to ``ASmoothMapEstimator`` and also
+adapted to work with ``MapDataset`` and ``MapDatasetOnOff``.
+
+To further improve API consistency the ``EnergyDispersion`` class
+was renamed to ``EDispKernel`` and the ``SkyModels``class was
```
Comment (Bultako) [inline CHANGES.rst]: ``SkyModels`` class

File: CHANGES.rst [hunk 8]
```diff
@@ -1,8 +1,116 @@
...
+
+Pull Requests
++++++++++++++
+
+This list is incomplete. Small improvements and bug fixes are not listed here.
+
+See the complete `Gammapy v0.16 merged pull requests list on Github <https://github.com/gammapy/gammapy/pulls?q=is%3Apr+milestone%3A0.16+is%3Aclosed>`__.
+
+
+
```
Comment (Bultako) [inline CHANGES.rst]: - [#2756] Add config params for get_flux_points method in High-level interface (José Enrique Ruiz)
- [#2747] Modify Config and Analysis to support SafeMaskMaker (Régis Terrier)

File: CHANGES.rst (hunk 1 above)
Comment (adonath) [reply CHANGES.rst]: Done

File: CHANGES.rst (hunk 3 above)
Comment (adonath) [reply CHANGES.rst]: I moved it up as the 3rd paragraph.

File: CHANGES.rst (hunk 4 above)
Comment (adonath) [reply CHANGES.rst]: Done

File: CHANGES.rst (hunk 5 above)
Comment (adonath) [reply CHANGES.rst]: Fixed

File: CHANGES.rst (hunk 6 above)
Comment (adonath) [reply CHANGES.rst]: Done

File: CHANGES.rst (hunk 2 above)
Comment (adonath) [reply CHANGES.rst]: Fixed

File: CHANGES.rst (hunk 7 above)
Comment (adonath) [reply CHANGES.rst]: Done

File: CHANGES.rst (hunk 8 above)
Comment (adonath) [reply CHANGES.rst]: Done

**Comment by adonath** [comment]
Thanks @registerrier and @Bultako for your comments!

**Pull request merged**